### PR TITLE
allow deletion of ghost comments

### PR DESCRIPTION
**Pull request by Trott**
@nodejs/tsc @nodejs/moderation

### CODE REVIEW

**Review by ronag — APPROVED** [review]

File: Moderation-Policy.md [hunk 1]
```diff
@@ -185,6 +185,9 @@ of the [Code of Conduct][].
   service advertising, etc.) are subject to immediate moderation.
 * Collaborators may use the Hide feature in the GitHub interface for off-topic
   posts by non-Collaborators.
+* Members of the Moderation Team or TSC can delete any issues or comments posted
+  by accounts that have been deleted by GitHub. These accounts show up in the
+  GitHub interface as user `ghost`.
```
Comment (Trott) [inline Moderation-Policy.md:190]:
```suggestion
  GitHub interface as user `ghost`. There is no need
  to screenshot or document these deletions.
```

**Review by richardlau — APPROVED** [review]

**Review by bnb — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by JohnTitor — APPROVED** [review]

**Review by ChALkeR — APPROVED** [review]

**Comment by ChALkeR** [comment]
This doesn't imply though that _all_ ghost comments _should_ be deleted.
As some might be helpful and/or make sense.

**Comment by ChALkeR** [comment]
This doesn't imply though that _all_ ghost comments _should_ be deleted.
As some might be helpful and/or make sense.

**Review by ljharb — APPROVED** [review]

**Review by targos — APPROVED** [review]

**Review by GeoffreyBooth — APPROVED** [review]

**Review by BethGriggs — APPROVED** [review]

**Review by gireeshpunathil — APPROVED** [review]

**Review by RaisinTen — APPROVED** [review]

**Comment by Trott** [comment]
With 11 of 21 TSC members endorsing this (if you include me), I'm going to land this. Thanks, everyone.

**Pull request merged**